freddytoolapi/Freddy.UnitTests/Utilities/CustomerViewBuilder.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Freddy.Application.Queries.Customers;
+using System;
+
+namespace Freddy.Application.UnitTests.API.Controllers.CustomerController
+{
+    public class CustomerViewBuilder
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public string Email { get; set; }
+        public string Phone { get; set; }
```

Comment:
Mas dovod aby to bolo public?

Reply:
z vonku sa properties nebudu pouzivat len cez With_whatever takze nie, opravim